{
  "uid" : "1ab2f919241ea4c5",
  "name" : "reverts when trying to transferFrom",
  "fullName" : "Contract: ERC721Pausable when token is paused.reverts when trying to transferFrom",
  "historyId" : "f7988b9086dea19e74d2f8b1239f2b1b",
  "time" : {
    "start" : 1635782738721,
    "stop" : 1635782739681,
    "duration" : 960
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04183d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04183d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/token/ERC721/extensions/ERC721Pausable.test.js:15:46)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "testClass",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "testMethod",
    "value" : "reverts when trying to transferFrom"
  }, {
    "name" : "package",
    "value" : "Contract: ERC721Pausable when token is paused"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "ec28a5dc453be21",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2871e80>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782613602,
        "stop" : 1635782614165,
        "duration" : 563
      }
    }, {
      "uid" : "5f62ade8f4861ff2",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fb4580>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782612430,
        "stop" : 1635782612991,
        "duration" : 561
      }
    }, {
      "uid" : "2991349739072cf7",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be9ea60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782612041,
        "stop" : 1635782612607,
        "duration" : 566
      }
    }, {
      "uid" : "40e15613dfbfb79",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a044d400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782594575,
        "stop" : 1635782595548,
        "duration" : 973
      }
    }, {
      "uid" : "81aafa77cf68d1e9",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0637130>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782585320,
        "stop" : 1635782585881,
        "duration" : 561
      }
    }, {
      "uid" : "f54720ff73fbd64a",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0683a60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782533064,
        "stop" : 1635782533626,
        "duration" : 562
      }
    }, {
      "uid" : "c4ebf4608b0fb6f5",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067ceb0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782521131,
        "stop" : 1635782521692,
        "duration" : 561
      }
    }, {
      "uid" : "782392ccff9b98d5",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a06376a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782518661,
        "stop" : 1635782519224,
        "duration" : 563
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "4d961897216da5ad",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/4d961897216da5ad",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635590408418,
          "stop" : 1635590425641,
          "duration" : 17223
        }
      }, {
        "uid" : "ffe790a2087c61a1",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/ffe790a2087c61a1",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be7aa90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635555013426,
          "stop" : 1635555014397,
          "duration" : 971
        }
      }, {
        "uid" : "1b55da495e2387f6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/1b55da495e2387f6",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551256189,
          "stop" : 1635551256249,
          "duration" : 60
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "1ab2f919241ea4c5.json",
  "parameterValues" : [ ]
}